.pre-commit-config.yaml

```diff
@@ -8,6 +8,17 @@ repos:
     -   id: mixed-line-ending
     -   id: trailing-whitespace
 
+-   repo: https://github.com/ikamensh/flynt/
+    rev: '0.55'
+    hooks:
+    -   id: flynt
+        pass_filenames: false
+        args: [
+            '--line-length 120',
+            '--transform-concats',
```

Comment:
I think this argument should be removed: it's available only on py3.8+ interpreters.

Reply:
But we are running our pre-commit on 3.8. I think it is an important option to have. It will just not work for local dev environments with an outdated python version. Will it crash or simply ignore that option? If the latter, I vote to keep it in, even though now pre-commit may succeed locally but then fail on GHA..... arrgh

Reply:
It crashes..

Reply:
But I think it's not actually _that_ important to have it in - we can transform the bulk of them by manual invocation of `flynt`.

Reply:
Fine, I cave ;) has been removed

Reply:
We could add `language_version: 'python3.8'` to the pre-commit config - that just means that `python3.8` is a hard requirement (to be installed on the machine, not necessarily that venv) for running the pre-commit hook.

Reply:
But then it won't run at all for devs without 3.8. Let's just omit it for now.